packages/dai-plugin-mcd/test/contracts/mockContracts.json

```diff
@@ -1,42 +1,42 @@
 {
-  "UNIV2DAIETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2DAIETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2DAIETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2DAIETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2WBTCETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2WBTCETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2WBTCETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2WBTCETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2USDCETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2USDCETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2USDCETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2USDCETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2DAIUSDC": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2DAIUSDC": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2DAIUSDC_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2DAIUSDC_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2ETHUSDT": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2ETHUSDT": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2ETHUSDT_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2ETHUSDT_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2LINKETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2LINKETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2LINKETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2LINKETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2UNIETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2UNIETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2UNIETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2UNIETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2WBTCDAI": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2WBTCDAI": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2WBTCDAI_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2WBTCDAI_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2AAVEETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2AAVEETH": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2AAVEETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2AAVEETH_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "UNIV2DAIUSDT": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "PIP_UNIV2DAIUSDT": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_JOIN_UNIV2DAIUSDT_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b",
-  "MCD_FLIP_UNIV2DAIUSDT_A": "0x6efDF221190D139DD29bdbC0aB984165C6cC6C4b"
+  "UNIV2DAIETH": "0xC226F3CD13d508bc319F4f4290172748199d6612",
```

Comment:
Also not familiar with all these changes, let's make sure we want them.

Reply:
I believe these are dummy addresses for tokens we're not supporting in the testchain yet.